### PR TITLE
completed javascript-bank challenge

### PR DESCRIPTION
**Pull request by daniel-sungwon-lee**
image

### CODE REVIEW

File: javascript-bank/account.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* exported Account */
+function Account (number,holder){
+  this.number = number
+  this.holder = holder
+  this.transactions=[]
+}
+
+Account.prototype.deposit=function(amount){
+  if (Math.sign(amount)===1){
+    this.transactions.push(amount)
```
Comment (shadorki) [inline javascript-bank/account.js]: Use the Transaction constructor function

File: javascript-bank/account.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    return true
+  } else {
+    return false
+  }
+}
+Account.prototype.withdraw=function(amount){
+  if (Math.sign(amount)===1){
+    this.transactions.push(-amount)
```
Comment (shadorki) [inline javascript-bank/account.js]: Use the Transaction constructor function

**Review by shadorki — CHANGES_REQUESTED** [review]

**Comment by daniel-sungwon-lee** [comment]
image

**Review by scbowler — APPROVED** [review]
Good job on the fix, looks good

**Pull request merged**